Mon Aug  4 19:57:46 BST 2008  Simon McVittie <simon.mcvittie@collabora.co.uk>
  * Use pedantically correct marshaller for a GArray, because GObject 2.17 actually checks for this :-(
diff -rN -u old-telepathy-gabble/src/presence-cache.c new-telepathy-gabble/src/presence-cache.c
--- old-telepathy-gabble/src/presence-cache.c	2008-08-05 07:18:44.578811391 +0100
+++ new-telepathy-gabble/src/presence-cache.c	2008-08-05 07:18:44.618813891 +0100
@@ -297,7 +297,7 @@
     G_SIGNAL_RUN_LAST,
     0,
     NULL, NULL,
-    g_cclosure_marshal_VOID__POINTER, G_TYPE_NONE, 1, DBUS_TYPE_G_UINT_ARRAY);
+    g_cclosure_marshal_VOID__BOXED, G_TYPE_NONE, 1, DBUS_TYPE_G_UINT_ARRAY);
   signals[NICKNAME_UPDATE] = g_signal_new (
     "nickname-update",
     G_TYPE_FROM_CLASS (klass),

